xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexTradeServiceRaw.java

```diff
@@ -190,7 +222,12 @@ private BitfinexOrderStatusResponse sendLimitOrder(
               isHidden,
               isPostOnly,
               ocoAmount);
-      response = bitfinex.newOrder(apiKey, payloadCreator, signatureCreator, request);
+      response =
+          decorateApiCall(
+                  () -> bitfinex.newOrder(apiKey, payloadCreator, signatureCreator, request))
+              .withRetry(retry("trade-limitOrder"))
```

Comment:
non-idempotent requests should not have retries on them, you don't want to mistakenly double send an order

Reply:
Thanks for this. I will remove the withRetry on the TradeService with @POST